src/viewport/size/browser-viewport-size.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from "@angular/core";
```

Comment:
since this is not exactly a sub-feature I prefer if it remains in the `/viewport` root